internal/pkg/cli/pipeline_status.go

```diff
@@ -0,0 +1,80 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+const (
+	pipelineStatusAppNamePrompt     = "Which application is the pipeline in?"
+	pipelineStatusAppNameHelpPrompt = "An application is a collection of related services."
+	pipelineStatusNamePrompt        = "Which pipeline's status would you like to show?"
+	pipelineStatusNameHelpPrompt    = "Displays the status of each of a pipeline's stages."
```

Comment:
Nit: I assume these will be for `Ask()`?  Since these aren't used yet I might remove these for now, but not a blocker.